src/broadcast.jl

```diff
@@ -135,6 +135,10 @@ end
     end
 end
 
+scalar_getindex(x) = x
+scalar_getindex(x::Ref) = x[]
+scalar_getindex(x::Tuple{<: Any}) = x[1]
```

Comment:
Is the `<:` necessary in a covariant parameter?

Reply:
No, shouldn't be, as `Tuple{<:Any} == Tuple{Any}`.

Reply:
True, I just made a habbit out of always using this pattern...